test/models/setting_test.rb

```diff
@@ -174,6 +174,13 @@ def test_settings_should_save_strings
     assert_equal "must be comma separated", setting.errors[:value].first
   end
 
+  test "trusted_hosts should accept values with spaces" do
```

Comment:
`trusted_hosts should trim values with spaces`

Reply:
https://www.betterspecs.org/#should says you should make it:
```suggestion
  test "trusted_hosts accepts values with spaces" do
```

Granted, that's for RSpec, but I like the convention.

Reply:
Thanks @ekohl , updated it